Modules/ContentKit/Resources/Content/curriculums/curriculum_leka_weather_robot-ED4C900D4ACB426384DC76F182BC19A5/activities/leka's_weather_1-2CDC8A5FBA2841159F6E7E2EB262FE7F.activity.yml

```
@@ -0,0 +1,200 @@
+# Leka - iOS Monorepo
+# Copyright APF France handicap
+# SPDX-License-Identifier: Apache-2.0
+
+version: 1.0.0
+uuid: 2CDC8A5FBA2841159F6E7E2EB262FE7F
+created_at: "2025-02-19T14:27:22.101000"
+last_edited_at: "2025-02-25T10:49:33.601149"
+name: leka's_weather_1
+status: published
+authors:
+  - leka
+skills:
+  - association
+  - association/logical
+tags:
+  - weather_elements
+hmi:
+  - tablet_robot
+types:
+  - one_on_one
+locales:
+  - en_US
+  - fr_FR
+l10n:
+  - locale: en_US
+    details:
+      icon: leka_weather_robot
+      title: Leka's Weather
+      subtitle: 2 images
+      short_description: >
+        A fun activity where the person accompanied associates objects with the
+        weather displayed by Leka by touching the correct images on the iPad.
+      description: Leka displays a weather forecast on its screen (sun, snow, night, hot or cold). The person accompanied must observe and touch the objects corresponding to this weather on the iPad.
+      instructions: "- Encourage the person accompanied to look at the Leka screen. - Ask him to observe the weather displayed. - Invite him to touch the corresponding objects on the iPad."
```

ici ça doit être sur plusieurs lignes

```suggestion
      instructions: |
      	- Encourage the person accompanied to look at the Leka screen.
      	- Ask him to observe the weather displayed.
      	- Invite him to touch the corresponding objects on the iPad.
```